frontend/src/pages/RecipientRecord/pages/components/CommunicationLogTable.scss

```diff
@@ -0,0 +1,19 @@
+@use "../../../../colors.scss" as colors;
+
+.ttahub-communication-log-table .usa-table {
+    margin: 0;
+}
+
+.ttahub-communication-log-table .usa-table thead th[aria-sort] {
+    background: transparent;
+}
+
+.ttahub-communication-log-table .usa-table thead th[aria-sort] button{
+    color: colors.$text-ink;
+    text-decoration: none;
+    font-weight: bold;
```

Comment:
bold v strong does it matter?

Reply:
```<strong>``` is HTML, just a preference to keep styles and content separate